src/block/block_compact.c

```diff
@@ -373,7 +365,7 @@ __block_compact_estimate_remaining_work(WT_SESSION_IMPL *session, WT_BLOCK *bloc
         __wt_verbose_debug2(session, WT_VERB_COMPACT,
           "%s: estimating -- pass %d: will rewrite %" PRIu64 " pages, next write offset: %" PRId64
           ", extra space: %" PRId64,
-          block->name, iteration, pages_to_move_orig, write_off, extra_space);
+          block->name, iteration, total_pages_to_move, write_off, extra_space);
```

Comment:
@pmacko86, please let me know if `pages_to_move_orig` was used on purpose here. I can see it was its only usage. I believe we should print what we will actually try to move, not what we could have moved in theory.

Reply:
Yes, the intention was to print the number of pages that will be moved in this pass, as opposed the total pages cumulative.

And yes, nice catch that there is a bug. I think it was supposed to be something like `pages_to_move_orig - pages_to_move`, which would give us the number of pages that could be moved just during the pass.

Reply:
Please see faf375f0d58543893bc48297eee5e0a6b84449be where I used `pages_to_move_orig - pages_to_move`.